cdm/src/main/java/ucar/nc2/ft2/coverage/simpgeometry/SimpleGeometryCoverage.java

```diff
@@ -194,16 +280,14 @@ public int getRank() {
     return getShape().length;
   }
 
-  //TODO make nonnull once we can access coordinate system builder

```

Comment:
It's good to see `getShape` and `getDimension` get implemented!

Reply:
Thanks! Yeah it's good having the CS builder under it.